### PR TITLE
Feature/import export zaaktypen

### PR DESCRIPTION
**Pull request by stevenbal**
fixes https://github.com/open-zaak/open-zaak/issues/214

### CODE REVIEW

File: src/openzaak/templates/admin/catalogi/import_zaaktype.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% extends "admin/base_site.html" %}
+{% load i18n %}
+
+{% block title %} {% trans "Import ZaakType" %} {{ block.super }} {% endblock %}
+
+{% block content %}
```
Comment (joeribekker) [inline src/openzaak/templates/admin/catalogi/import_zaaktype.html]: I gave a proper example after redoing some layout for catalogus, can you add it here too,

File: src/openzaak/components/catalogi/admin/catalogus.py [hunk 2]
```diff
@@ -80,17 +101,176 @@ class CatalogusAdmin(
...
+                            request,
+                            _("ZaakType successfully imported"),
+                            level=messages.SUCCESS,
+                        )
+                        return HttpResponseRedirect(
+                            reverse(f"admin:catalogi_catalogus_changelist")
+                        )
+                    except CommandError as exc:
+                        self.message_user(request, exc, level=messages.ERROR)
+        elif "_select" in request.POST:
```
Comment (joeribekker) [inline src/openzaak/components/catalogi/admin/catalogus.py]: Hmm, can you split it up in 2 views?

And move this whole inline view "thing" to a nice `admin_views` file and `ImportZaaktypeStep1View` and `ImportZaaktypeStep2View` class. and make it a class while at it. In `get_urls` you can reference it like this: `self.admin_site.admin_view(ImportZaaktypeView.as_view()))`

File: src/openzaak/components/catalogi/admin/mixins.py [hunk 3]
```diff
@@ -219,8 +164,9 @@ def response_post_save_change(self, request, obj):
             resource_list, id_list = self.get_related_objects(obj)
 
             response = HttpResponse(content_type="application/zip")
+            filename = str(obj).replace(" ", "")
```
Comment (joeribekker) [inline src/openzaak/components/catalogi/admin/mixins.py]: use `slugify` to turn an unsafe name into a safe one.

File: src/openzaak/components/catalogi/admin/mixins.py [hunk 4]
```diff
@@ -268,3 +216,42 @@ def get_extra_context(self, request, *args, **kwargs):
...
+            path(
+                "import/",
+                self.admin_site.admin_view(self.import_view),
+                name=f"catalogi_{self.resource_name}_import",
+            )
+        ]
+        return my_urls + urls
+
+    def import_view(self, request):
+        form = CatalogusImportForm(request.POST, request.FILES)
```
Comment (joeribekker) [inline src/openzaak/components/catalogi/admin/mixins.py]: Woops, I just merged the other PR that changed this file.

File: src/openzaak/conf/includes/base.py [hunk 5]
```diff
@@ -314,6 +314,7 @@
 ]
 
 SESSION_COOKIE_NAME = "openzaak_sessionid"
+SESSION_ENGINE = "django.contrib.sessions.backends.cache"
```
Comment (sergei-maertens) [inline src/openzaak/conf/includes/base.py]: Is this needed?

Comment (joeribekker) [reply src/openzaak/conf/includes/base.py]: @stevenbal you prolly discussed offline but why was this needed?

**Pull request merged**